MHCollapsibleViewManager.m

```diff
@@ -0,0 +1,228 @@
+//
+//  MHCollapsibleViewManager.m
+//  This is a manager of a uiview and view controller
+//
+//  Created by Lizzy Randall on 5/6/15.
+//  Copyright (c) 2015 Lizzy Randall. All rights reserved.
+//
+
+#import "MHCollapsibleViewManager.h"
+#import "MHFilterLabel.h"
+
+@implementation MHCollapsibleViewManager
+
+//@synthesize tableView;
+//@synthesize viewController;
+@synthesize rowAnimation;
+
+@synthesize data;
+@synthesize cellStyles;
+@synthesize headerTitle;
+
+@synthesize expanded;
+@synthesize numOfRows;
+
+//Initialize with animation, title and tableView
+- (id)initalizeManager:(UITableViewRowAnimation)animation title:(NSString*) title tableView:(UITableView*)tableView
+{
+    self.headerTitle = title;
+    self.rowAnimation = animation;
+    //initialized first and then can be overwritten by controller
+    //tableView is needed to create cells to store types
+    [self setCellProperties:nil header:nil clickedCell:nil normalCell:nil tableView:tableView];
```

Comment:
In the body of an initializer you need to call the base initializer - (instancetype)init; (which every class inherits from its parent. You should also look to put as much of the initialization of the object as possible into the base initializer. In this case all the initialization needs to be done in this method but if you wanted to initialize the data array before calling initializeData: you would do that in init.
eg
- (instancetype)initWithAnimation:(UITableViewRowAnimation)animation title:(NSString) title tableView:(UITableView)tableView {
    self = [self init];
    if (self) {
        self.headerTitle = title;
        self.rowAnimation = animation;
        //initialized first and then can be overwritten by controller
        //tableView is needed to create cells to store types
        [self setCellProperties:nil header:nil clickedCell:nil normalCell:nil tableView:tableView];
    }

    return self;
}

- (instancetype)init {
    self = [super init]; //if you override a method you need to manually call it's parent's version of the method. For most methods this should be done first.
    if (self) {
        self.data = [NSMutableArray array];
    }

    return self;
}
}